File: solution/02xxx/2720 - 세탁소 사장 동혁/2720.c
#include <stdio.h>

int main() {
    int n, money;
    scanf("%d", &n);
    for(int i = 0; i < n; i++) {
        scanf("%d", &money);
        printf("%d ", money/25);
        money -= money/25*25;
        printf("%d ", money/10);
        money -= money/10*10;
        printf("%d ", money/5);
        money -= money/5*5;
        printf("%d\n", money);
    }
}

File: solution/02xxx/2720 - 세탁소 사장 동혁/2720.py
import sys
input = sys.stdin.readline

t = int(input())

for _ in range(t):
    money = int(input())

    print(money // 25, end=" ")
    money -= money // 25 * 25
    print(money // 10, end=" ")
    money -= money // 10 * 10
    print(money // 5, end=" ")
    money -= money // 5 * 5
    print(money)